test/tools/libtesteth/ImportTest.cpp

```diff
@@ -182,15 +182,14 @@ bytes ImportTest::executeTest()
 			testObj["blocks"] = blocksArr;
 			json[testname] = testObj;
 
-			//Write a filler file to the filler folder
+			// Write a filler file to the filler folder
 			BCGeneralStateTestsSuite genSuite;
 			fs::path const testFillerFile = genSuite.getFullPathFiller(TestOutputHelper::caseName()) / fs::path(testname + "Filler.json");
 			writeFile(testFillerFile, asBytes(json_spirit::write_string((mValue)json, true)));
 
-			//Write a blockchain test .json file from dynamic filler 'json' that is produced from state test filler
-			fs::path const testFile = genSuite.getFullPath(TestOutputHelper::caseName()) / fs::path(testname + ".json");
-			json_spirit::mValue const generatedJson = genSuite.doTests((json_spirit::mValue)json, true);
-			writeFile(testFile, asBytes(json_spirit::write_string(generatedJson, true)));
+			// Execute test filling for this file
+			genSuite.executeTest(TestOutputHelper::caseName(), testFillerFile);
+
```

Comment:
Why has `writeFile(testFile)` disappeared?

Reply:
executeTest perform checks on Filler file name, remove //comments from it, and adds a client infrormation about the test

Reply:
That does not sound like executing a test.

Reply:
well it actually does execute a specified filler file to produce a test. 
I could have just create a filler file here and thats it. but with this one line of code I don't have to run two separate testeth commands in order to create a blockchain test from state test

Reply:
actually I wanted to make executeTest private. it is just too convenient to have blockchain tests generated by one command instead of two.

Reply:
> actually I wanted to make executeTest private.

Make small changes until that's possible.  Don't try to make it private in the first PR.

Reply:
Why has `writeFile(testFile)` disappeared?

Reply:
this logic is inside executeTest